src/Microsoft.AspNet.Mvc.Xml/Constants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    internal static class Constants
```

Comment:
the typical guidance we follow is we do not create internal classes ... @yishaigalatzer whats your call?